docs/configuration/backends/kubernetes.md

```diff
@@ -218,7 +218,7 @@ infos:
     serialnumber: true
 ```
 
-If `pem` is set, it will add a `X-Forwarded-Tls-Client-Cert` header that contains the escaped pem as value.
+If `pem` is set, it will add a `X-Forwarded-Tls-Client-Cert` header that contains the escaped pem as value. (Note this works only with certificates issued by CAs included in the applicable [EntryPoint ClientCA section](/configuration/entrypoints/#tls-mutual-authentication); certificates from other CAs are not passed through as-is.)
```

Comment:
As this mechanism applies to the both options, this note could be added after the next sentence and start with something like "These two options applies only...". WDYT ?

Reply:
Good catch. I've updated the branch accordingly.